libass/ass_drawing.c

```diff
@@ -204,15 +199,12 @@ static inline void translate_point(ASS_Drawing *drawing, FT_Vector *point)
 }
 
 /*
- * \brief Evaluate a curve into lines
- * This curve evaluator is also used in VSFilter (RTS.cpp); it's a simple
- * implementation of the De Casteljau algorithm.
```

Comment:
Is this comment no longer true?

Reply:
Yes, that's no longer true and for quite long time at that.